package.nuspec

```diff
@@ -9,8 +9,27 @@
         <repository type="git" url="$repoUri$" commit="$commitId$"/>
     </metadata>
     <files>
-        <file src="packages/**/*.*" />
-        <file src="scripts/**/*.*" />
-        <file src="*.*" />
+      <!-- iOS device debug -->
```

Comment:
Why can't you capture all these files via glob patterns? You can use excludes if some artifacts are undesired.

Reply:
See @anandrajeswaran 's comment [here](https://github.com/microsoft/fluentui-react-native/pull/529#discussion_r509911726). We preferred being explicit